src/BloomExe/web/ServerBase.cs

```
@@ -326,16 +311,13 @@ private static void AddUrlAccessControlEntry()
 			Process.Start(startInfo);
 		}
 
-		private static bool GetIsAbleToUsePort()
+		private static void GetIsAbleToUsePort()
 		{
-			try
-			{
-				var x = new WebClientWithTimeout { Timeout = 3000 };
-				return ("OK" == x.DownloadString(PathEndingInSlash + "testconnection"));
-			}
-			catch (Exception e)
+			var x = new WebClientWithTimeout { Timeout = 3000 };
+			if ("OK" != x.DownloadString(PathEndingInSlash + "testconnection"))
```

Don't ignore the exception here.